conans/client/command.py

```diff
@@ -124,7 +124,57 @@ def new(self, *args):
                         upload_url=args.ci_upload_url,
                         gitlab_gcc_versions=args.ci_gitlab_gcc,
                         gitlab_clang_versions=args.ci_gitlab_clang)
+                        
 
+    def pack(self, *args):
+        """ Packs a binary package.
+        """
+        parser = argparse.ArgumentParser(description=self.pack.__doc__, prog="conan pack")
+        parser.add_argument("reference", help='user/channel, or a full package reference'
+                                              ' (Pkg/version@user/channel), if name and version '
+                                              ' are not declared in the recipe')
+
+        parser.add_argument("--package_id", default=None, required=True, help="specify package id to pack.")
+
+        parser.add_argument("--out", default=os.getcwd(), help='Directory to output the packed binaries.')
+
+        parser.add_argument("--packname", default=None, help='Name used for packing. '
+                                                             'If ommited it will be automatically generated.')
+
+        # TODO: obtain the list of supported methods from the packing class
+        parser.add_argument("--target", default='zip',
+                            help='Scheme to use for packing the binary package.')
+
+        args = parser.parse_args(*args)
+
+        try:
+            reference = ConanFileReference.loads(args.reference)
+        except ConanException:
+            print("Could not load reference: %s" % str(args.reference))
```

Comment:
Use functions from the [Conan output system](https://github.com/conan-io/conan/blob/develop/conans/client/output.py) to print messages. Or use the `ConanException`

Reply:
this was a conan-independent tool. I will switch it eventually, thx.